include/xcp-ng/generic/io.h

```diff
@@ -18,6 +18,7 @@
 #define _XCP_NG_GENERIC_IO_H_
 
 #include <poll.h>
+#include <sys/uio.h>
```

Comment:
Can't you do a forward declaration and include in the .c ?

Reply:
Yes I can.